docs/api/websocket.md

```diff
@@ -525,3 +525,28 @@ If an error occurs, the `success` key in the `result` message will be set to `fa
    }
 }
 ```
+
+### Error handling during service calls and translations
+
+When a `ServiceValidationError` (`service_validation_error`) exception was raised during a websocket service call. A stack trace is printed to the logs at debug level only. Raising `HomeAssistantError` (`home_assistant_error`) or subclass exception will still print a full stack trace to the logs.
```

Comment:
I think we should talk about how the exceptions are handled not how they are raised. We're documenting the websocket API.